cluster/gce/util.sh

```diff
@@ -653,14 +653,14 @@ function kube-down {
     "${NODE_INSTANCE_PREFIX}-group")
   if [[ "$deleteCmdOutput" != ""  ]]; then
     # Managed instance group deletion is done asyncronously, we must wait for it to complete, or subsequent steps fail
-    deleteCmdOperationId=$(echo $deleteCmdOutput | grep "Operation:" | sed "s/.*Operation:\s//" | sed "s/\s.*//" | sed "s/ //g")
+    deleteCmdOperationId=$(echo $deleteCmdOutput | grep "Operation:" | sed "s/.*Operation:[[:space:]]*\([^[:space:]]*\).*/\1/g")
     if [[ "$deleteCmdOperationId" != ""  ]]; then
       deleteCmdStatus="PENDING"
       while [[ "$deleteCmdStatus" != "DONE" ]]
       do
```

Comment:
the indentation under the do is inconsistent (spaces/tabs) which my editor renders differently than github (way worse).